src/Microsoft.Diagnostics.Runtime.Utilities/ClrObjectExtensions.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+using Microsoft.Diagnostics.Runtime;
+
+namespace Microsoft.Diagnostics.Runtime.Utilities
+{
+    public static class ClrObjectExtensions
+    {
+        public static string GetStringValue(this ClrObject clrObject, int maxLength = int.MaxValue)
```

Comment:
Why not use `ClrObject.AsString`?  We probably need to add `ClrObject.IsString` but can't this extension method be replaced with:  `clrObject.Type.IsString ? clrObject.AsString(maxLength) : null` ?

See: https://github.com/microsoft/clrmd/blob/17bea31ca88e89ec695b2b2073eb9b61cc425e6e/src/Microsoft.Diagnostics.Runtime/src/Common/ClrObject.cs#L334

Reply:
You are right. I migrated my own library to ClrMD 2.0 few weeks ago and I probably missed that.
It can still be useful to avoid the check on the type. Do you want me to change the extension method name?